README.md

```diff
@@ -73,119 +73,140 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
-
-### Mouse Utilities
-
-- Changed the opacity setting to the 1-100 range. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Added a scrollbar to the additional lines dialog. Thanks [@davidegiacometti](https://github.com/davidegiacometti)! (This was a hotfix for 0.64)
+- Updated the plus icon. Thanks [@niels9001](https://github.com/niels9001)! (This was a hotfix for 0.64)
+- Prevent the new entry content dialog from overlapping the title bar.
+- Updated the name for the additional lines feature. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Added a workaround for an issue causing the context menu not opening on right-click. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 
 ### PowerToys Run
 
-- Changed image loading to release the images in PowerToys Run main executable. This is a try to fix the "app.dark.png" missing issues received after a PowerToys update.
-- Fixed the PowerToys Run hiding after the default action failed. Thanks [@hlaueriksson](https://github.com/hlaueriksson)!
-- Fixed the PowerToys Run allows showing after a context menu action succeeded. Thanks [@hlaueriksson](https://github.com/hlaueriksson)!
+- Added pinyin support to the search. Thanks [@frg2089](https://github.com/frg2089)!
+- Fixed an error in the TimeZone plugin preventing searching for standard time zones. Thanks [@Tantalus13A98B5F](https://github.com/Tantalus13A98B5F)!
+- Added the English abbreviations as fallbacks in the UnitConverter plugin. Thanks [@Tantalus13A98B5F](https://github.com/Tantalus13A98B5F)!
 
 ### Quick Accent
 
-- Corrected "Dutch" word to "German". Thanks [@damienleroy](https://github.com/damienleroy)!
-- Added the Portuguese language accents. Thanks [@pcanavar](https://github.com/pcanavar)!
-- Fixed positioning of toolbar on scaled desktops.
+- Added mappings for the mu, omicron, upsilon and thorn characters.
+- Added a setting to exclude apps from activating Quick Accent.
+- Fixed an issue causing the selector to trigger when leaving the lock screen. Thanks [@damienleroy](https://github.com/damienleroy)!
+- Added the Croatian, Netherlands, Swedish and Welsh character sets. Thanks [@damienleroy](https://github.com/damienleroy)!
+- Added support for more unicode characters. Thanks [@char-46](https://github.com/char-46)!
+- Shift-space can now navigate backwards in the selector. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Added the Catalan accented characters. Thanks [@ivocarbajo](https://github.com/ivocarbajo) and [@codingneko](https://github.com/codingneko)!
+- Added the Kurdish accented characters.
+- Added the Serbian accented characters. Thanks [@damienleroy](https://github.com/damienleroy)!
+- Added the Irish and Scottish accented characters.
+- Added the description for the currently selected character in the selector.
+- Fixed a bug causing the selector window to appear blank.
```

Comment:
We should have one "Added xyz language" list for each contributor and not multiple for  each one.

Who added the languages without contributor name? The PT core team?

Reply:
Those came in different PRs, that's why I ended up separating them.
The ones without attribution were by the core team. (They were asked by the community, though)